libs/template/templates/default-python/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,32 @@
+bundle:
+  name: com.mycompany.{{.project_name}}
```

Comment:
do we really want this Javaesque prefix? IMHO we should just strip this.

Reply:
We need some kind of unique name to avoid collisions in target workspace dir. So far we did that with qualified names. But they only help so much.

A different approach we could take here is to add some kind of deterministic unique id for the workspace folder. Or to have some form of collision detection.